DESCRIPTION.md

```diff
@@ -12,6 +12,8 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
   - Added easy logging configuration so that users can easily generate log file by setup log config in `$SNOWFLAKE_HOME/config.toml`.
   - Improved s3 acceleration logic when connecting to China endpoint.
+  - Added support for structured types to fetch_pandas_all
```

Comment:
actually I realize this can be a breaking change to customer.
for example, for a column of Object type, previously we get json string, but now it's the python object so users have to rewrite their code?

are you aware if this is part of a snowflake BCR?